tests/test_backend.py

```diff
@@ -49,175 +51,185 @@ def test_node_info_cache(self, get_cluster_info):
         backend.set('key2', 'val')
         backend.get('key2')
         backend._class.assert_called_once()
-    eq_(backend._class.call_args[0], (servers,))
+
+        eq_(backend._class.call_args[0], (servers,))
         assert backend._cache.get.call_count == 2
         assert backend._cache.set.call_count == 2
 
         get_cluster_info.assert_called_once()
 
-    @patch.object(ConfigurationEndpointClient, 'get_cluster_info')
-    def test_failed_to_connect_servers(self, get_cluster_info):
-        from django_elastipymemcache.backend import ElastiPymemcache
-        backend = ElastiPymemcache('h:0', {})
-        get_cluster_info.side_effect = OSError()
-        assert backend.get_cluster_nodes() == []
 
-    @patch.object(ConfigurationEndpointClient, 'get_cluster_info')
-    def test_invalidate_cache(self, get_cluster_info):
-        from django_elastipymemcache.backend import ElastiPymemcache
-        servers = [('h1', 0), ('h2', 0)]
-        get_cluster_info.return_value = {
-            'nodes': servers
-        }
+@patch.object(ConfigurationEndpointClient, 'get_cluster_info')
+def test_failed_to_connect_servers(self, get_cluster_info):
```

Comment:
Did you intend to take this method (and all following test methods) out of the class?

If so, it seems quite weird to leave the `self` argument

(the change here is combined -- your merge from master put the method in the class, and you then took it out)